doc/api/util.md

```diff
@@ -1422,6 +1426,9 @@ changes:
     * `default` {string | boolean | string\[] | boolean\[]} The default option
       value when it is not set by args. It must be of the same type as the
       `type` property. When `multiple` is `true`, it must be an array.
+    * `required` {boolean} Whether the argument must be supplied. **Default:** `false`.
+    * `placeholder` {string | boolean} The default value to supply when an
```

Comment:
The approach taken for `default` is:

> The default option value when it is not set by args. It must be of the same type as the `type` property. When `multiple` is `true`, it must be an array.

The reasoning is to make it simple and consistent for the author. If a value is returned, it is of a consistent type.

Reply:
Ah, placeholder does not need to allow array for returned option value to be array (for 'multiple'). I was on wrong track, can resolve this comment.